tests/bluetooth/ctrl_isoal/Kconfig

```diff
@@ -12,6 +12,21 @@ config BT_CTLR_CONN_ISO_GROUPS
 	range 1 240
 	default 1
 
+config BT_CTLR_ISOAL_LOG_LEVEL
+	int "ISO-AL logging level"
+	range 0 4
+	default 3
+	help
+	  Use this option to enable ISO-AL logging at the given level.
+	  Default is LOG_LEVEL_INF (3).
+
+config BT_CTLR_ISOAL_LOG_DBG_VERBOSE
+	bool "ISO-AL verbose debug logging"
+	depends on BT_CTLR_ISOAL_LOG_LEVEL = 4
+	default n
+	help
+	  Use this option to enable ISO-AL verbose debug logging.
```

Comment:
Why do we need to define this in two places?

Reply:
Some of the configurations used in the ISO-AL unit tests have a long chain of dependencies. Redefining it is easier for the unit test configuration rather than trying figure out the chain and trying to fulfill the whole chain.